pkg/execution/state/redis_state/queue.go

```diff
@@ -984,7 +1000,7 @@ func (q *queue) ItemPartitions(ctx context.Context, i QueueItem) ([]QueuePartiti
 		fnPartition.ConcurrencyLimit = limits.AccountLimit
 	}
 
-	if len(ckeys) > 0 {
+	if enableKeyQueues && len(ckeys) > 0 {
```

Comment:
We decided to roll back key queues. This line makes sure we only ever return the default partition, and no concurrency key queues.